ui/src/components/SkipNavigation/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import { Typography } from '@material-ui/core';
+import Link from '../../utils/Link';
+
+export default class SkipNavigation extends Component {
+  render() {
+    const { href, text } = this.props;
+
+    return (
+      <Link to={href}>
```

Comment:
I think you can use a Button from `src/components/Button` instead and have an `onClick` that jumps to the first focusable element inside `<main>`.

Reply:
Alright. Will do.

Reply:
I have done these changes.